test/runnable/varargstringlit.d

```diff
@@ -0,0 +1,19 @@
+extern(C) void vf(int x, ...)
+{
+    import core.stdc.stdarg;
+    va_list args;
+
+    va_start(args, x);
+    const(char)* s;
+    va_arg(args, s);
+    assert(s[0..2] == "hi");
+}
+
+unittest
```

Comment:
Any reason you don't use `void main()` here?
`unittest` blocks don't automatically get run here.

Reply:
Oops, I have my editor set up to pass `-unittest -main`, and got in the habit of doing that instead. Fixed.

Reply:
FYI you can do `./run.d runnable/varargstringlit.d` (almost as good)